### DISCUSSION TITLE
Can not cancel overlay popups

### DISCUSSION THREAD

**Discussion by maggigisla** [Kong/insomnia#4791] [Help]
Hi ...
Time and again when using Insomnia there is a overlay popup windows that "pop's up" and I can not get it to go away unless I select some item in the popup window, I try to press Esc, Tab or other keys but it does not go away unless I select something. Then it puts my selection in front of url in address bara or any field that had focus at that time.
Has anyone experienced this and know how to cancel the popup ?
See photo ..
![overlay](https://user-images.githubusercontent.com/11015703/169049813-2e48f8be-8c3a-43dd-820a-9fdc1444d97f.png)
.

**Comment by Calabrone76** [comment]
I have the [same problem](https://github.com/Kong/insomnia/discussions/5110), have you found a solution?

**Comment by maggigisla** [comment]
No, still the same
